logstash-core/spec/logstash/compiler/compiler_spec.rb

```diff
@@ -49,103 +49,6 @@ def rand_meta
     end
   end
 
-  describe "compile with empty source" do
```

Comment:
are these specs really to be "lost" here, seems like they test the Java logic, right?
the stuff behind `ConfigCompiler.compileSources(source_with_metadata, support_escapes)`

Reply:
oh I see, scratch that, you actually moved all of these into Java. nice job!